packages/core-accountsjs/src/director/Oauth2Adapter.ts

```diff
@@ -0,0 +1,40 @@
+/* eslint-disable no-unused-vars */
+/* eslint-disable @typescript-eslint/no-unused-vars */
+import { log, LogLevel } from '@unchainedshop/logger';
+import { IOauth2Adapter } from '@unchainedshop/types/accounts.js';
+import { Oauth2Error } from './Oauth2Error.js';
+
+export const Oauth2Adapter: Omit<IOauth2Adapter, 'key' | 'label' | 'version' | 'orderIndex'> = {
+  provider: '',
+  config: null,
+
+  actions: ({ redirectURL }) => {
+    return {
+      configurationError: () => {
+        throw new Error(Oauth2Error.NOT_IMPLEMENTED);
+      },
+      isActive: () => {
+        return true;
+      },
+      getAuthorizationCode: async (authorizationCode: string) => {
+        return null;
+      },
+      getAccountData: async (userAuthorizationToken: any) => {
+        return null;
+      },
+      isTokenValid: async (token) => {
```

Comment:
How would you suggest using isTokenValid and when? How does this influence the login with the user, the session token?